src/wasm-stack.h

```diff
@@ -93,17 +93,16 @@ class BinaryInstWriter : public OverriddenVisitor<BinaryInstWriter> {
   BinaryInstWriter(WasmBinaryWriter& parent,
                    BufferWithRandomAccess& o,
                    Function* func,
-                   bool sourceMap,
                    bool DWARF)
-    : parent(parent), o(o), func(func), sourceMap(sourceMap), DWARF(DWARF) {}
+    : parent(parent), o(o), func(func), DWARF(DWARF) {}
 
   void visit(Expression* curr) {
-    if (func && !sourceMap) {
-      parent.writeDebugLocation(curr, func);
+    if (func) {
+      parent.trackExpressionStart(curr, func);
```

Comment:
`track` sounds like something we would do in a reader, but this is the writer. Maybe `note` or `visit` would be better?

Reply:
`track` vs `note` seem very similar - maybe some nuance I am missing? The meaning is to update some data structure (about the binary location we wrote at).

We do already have `binaryLocationTrackedExpressionsForFunc`, which `track` is consistent with. E.g. `trackExpressionStart` works on `binaryLocationTrackedExpressionsForFunc`.

Reply:
I see, I didn't realize that this was for updating a data structure at the time. This makes more sense now.